app.py

```diff
@@ -38,6 +38,7 @@ def sign_up():
 		mail.send(message)
 	except Exception as e:
 		# would be great to have logs here
+		print("Form could not load, server error!")
```

Comment:
We need to throw an error back to the client here. Probably something like:
```
return jsonify(success=false)
```

You'll also need to change the `finally` below to an `else`